src/vsfilesystem.cpp

```diff
@@ -602,23 +602,31 @@ void InitDataDirectory()
     if ( !vegastrike_cwd.empty() ) {
         data_paths.push_back( vegastrike_cwd );
         data_paths.push_back( vegastrike_cwd+"/.." );
-        data_paths.push_back( vegastrike_cwd+"/../data4.x" );
-        data_paths.push_back( vegastrike_cwd+"/../../data4.x" );
+        // DELETE 4.x no longer used since .5 release obsolete, removal required.
+        // data_paths.push_back( vegastrike_cwd+"/../data4.x" );
+        // data_paths.push_back( vegastrike_cwd+"/../../data4.x" );
+        // data_paths.push_back( vegastrike_cwd+"/data4.x" );
         data_paths.push_back( vegastrike_cwd+"/../../data" );
-        data_paths.push_back( vegastrike_cwd+"/data4.x" );
         data_paths.push_back( vegastrike_cwd+"/data" );
         data_paths.push_back( vegastrike_cwd+"/../data" );
         data_paths.push_back( vegastrike_cwd+"/../Resources" );
+        data_paths.push_back( vegastrike_cwd+"/Assets-Production");
+        data_paths.push_back( vegastrike_cwd+"/../Assets-Production");
+        data_paths.push_back( vegastrike_cwd+"/../../Assets-Production");
```

Comment:
Why did you add these?

Reply:
If I understand correctly, [vegastrike/Assets-Production](https://github.com/vegastrike/Assets-Production) is the GitHub equivalent to the `data`/`hqtextures` folder on SourceForge. When I clone it to my local machine, it ends up in a local folder that is also called `Assets-Production` by default.

I figured others would encounter the same behavior. Not only that, but when I tried configuring the data directory using `ccmake`, it didn't like it **at all**. I tried several different ways of specifying that path, but none of them worked.